src/next.md

```diff
@@ -66,27 +66,48 @@ test/lib-test/ProblemList.txt
 Use the suitable ProblemList and add a line like this in the proper section:
 
 ~~~
-sun.tools.jcmd.MyTest.java                        4711   windows-all
+foo/bar/MyTest.java                        4711   windows-all
 ~~~
 
 In this example, `MyTest.java` is ProblemListed on windows, tracked by bug `JDK-4711`.
 
 Currently there's [no support for multiple lines for the same test](https://bugs.openjdk.java.net/browse/CODETOOLS-7902481). For this reason it's important to always make sure there's no existing entry for the test before adding a new one, as multiple entries might lead to unexpected results, e.g.
 
 ~~~
-sun.tools.jcmd.MyTest.java                        4710   generic-all
+foo/bar/MyTest.java                        4710   generic-all
 ...
-sun.tools.jcmd.MyTest.java                        4711   windows-all
+foo/bar/MyTest.java                        4711   windows-all
 ~~~
 
 This would lead to `sun.tools.jcmd.MyTest.java` being ProblemListed only on `windows-all`. The proper way to write this is:
 
 ~~~
-sun.tools.jcmd.MyTest.java                        4710,4711   generic-all,windows-all
+foo/bar/MyTest.java                        4710,4711   generic-all,windows-all
 ~~~
 
 Although `windows-all` isn't strictly required in this example, it's preferable to specify platforms for each bugid (unless they are all `generic-all`), this makes it easier to remove one of the bugs from the list.
 
+#### ProblemListing some, but not all, test cases in a file
```

Comment:
actually, due to [CODETOOLS-7902712](https://bugs.openjdk.java.net/browse/CODETOOLS-7902712), you can't use `foo/bar/MyTest.java` entry to exclude all test cases, you have to list all of them explicitly.

Reply:
Thanks for catching that! I've added a comment about this.